### PR TITLE
Develop 6

### PR DESCRIPTION
**Pull request by chedman**
Q1 and Q2 recode

### CODE REVIEW

File: Prog/Q1 and Q2 recode.do [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+* Environment: Windows:
+* Description: 
+*
+* Modifications: 
+*/
+**Use raw open eneded file
+use "L:\Libraries\Voices\Raw\Q1.DTA", clear
+rename CaseID caseid
+
+**Merge recode dataset
```
Comment (ptatian) [inline Prog/Q1 and Q2 recode.do]: I would add a comment to the program here to indicate that this data set comes from the Excel file that contains your manual recoding.

**Comment by ptatian** [comment]
@chedman Somewhat similar issue here as with issue #8, although this one was intended to be more of a stand-alone process. We need to create a permanent SAS data set on DCDATA with the recoded responses. That means a data set with 1 row per survey respondent with the Q1 & Q2 recode flags (dc, mc, pg, etc., but I would rename these with the Q1_ or Q2_ prefix in the final data set) as well as additional dummy variables we would use to create the tables (which I'm not seeing yet). 

Again, let's make use of what you've done in STATA, but we'll need a new SAS program that is going to import that information and create a final data set on L:. To create final data sets, first create a temporary (let's name it **Voices_2017_q1_q2_recode**) SAS data set that has everything you need. Then, use the %Finalize_data_set() macro to create the final version.

```
%Finalize_data_set( 
  data=Voices_2017_q1_q2_recode,
  out=Voices_2017_q1_q2_recode,
  outlib=Voices,
  label="VoicesDMV 2017 survey, questions Q1 and Q2, recode",
  sortby=caseid,
  revisions=%str(New file.)
)
```

The permanent data set won't be until you run the final batch submit of the program on L:, which you should not do until the program has been reviewed and approved.

Let me know if you have any questions.

**Comment by chedman** [comment]
@ptatian Sounds good. I'll get started addressing this issue as well.

**Comment by lhendey** [comment]
@chedman  can you correct the labels on Voices_q1_q2_recode.sas to indicate what the question is? still reviewing the stata code.

**Comment by lhendey** [comment]
@chedman - sync first.

**Comment by chedman** [comment]
@lhendey yeah - can do.

**Comment by chedman** [comment]
@lhendey I just made the updates and committed the changes.

**Comment by lhendey** [comment]
@chedman can you batch submit the sas program for this one on L? I did additional coding in stata to reduce the number of no place identified.

**Comment by chedman** [comment]
@lhendey Yeah, I'll sync and do that now

**Pull request merged**